tools/notification-configuration/common/Services/AzureDevOpsService.cs

```diff
@@ -91,7 +91,18 @@ public async Task<IEnumerable<BuildDefinition>> GetPipelinesAsync(string project
         public async Task<BuildDefinition> GetPipelineAsync(string projectName, int pipelineId)
         {
             var client = await GetClientAsync<BuildHttpClient>();
-            return await client.GetDefinitionAsync(projectName, pipelineId);
+            BuildDefinition result;
```

Comment:
Do we need to clean up notification groups in this case?

Reply:
Agreed. This is coming in another PR.